src/site/markdown/JDK_MANAGEMENT.md

```diff
@@ -0,0 +1,597 @@
+jdkDistribution=temurin      # Disco API distribution name
+
+# Optional: Update policy (Maven-style)
+jdkUpdatePolicy=daily        # never, daily, always, interval:X
+
+# Optional: Direct URL (overrides version/distribution resolution)
+jdkDistributionUrl=https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.7%2B7/OpenJDK17U-jdk_x64_linux_hotspot_17.0.7_7.tar.gz
+
+# Optional: SHA-256 checksum for security
+jdkSha256Sum=aee68e7a34c7c6239d65b3bfbf0ca8f0b5b5b6e8e8e8e8e8e8e8e8e8e8e8e8e8
+
+# Optional: Force re-download
+alwaysDownloadJdk=false
+```
+
+### Toolchain JDK Configuration
+
+For multi-JDK builds using Maven toolchains:
+
+```properties
+# Toolchain JDK (automatically added to toolchains.xml)
+toolchainJdkVersion=11
+toolchainJdkDistribution=temurin      # Disco API distribution name
+
+# Optional: Direct URL (overrides version/distribution resolution)
+toolchainJdkDistributionUrl=https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.19%2B7/OpenJDK11U-jdk_x64_linux_hotspot_11.0.19_7.tar.gz
+
+# Optional: SHA-256 checksum for security
+toolchainJdkSha256Sum=aee68e7a34c7c6239d65b3bfbf0ca8f0b5b5b6e8e8e8e8e8e8e8e8e8e8e8e8e8
+```
+
+## JDK Version Syntax
+
+The `jdkVersion` and `toolchainJdkVersion` properties support flexible version formats via Foojay Disco API integration:
+
+### Major Version (Recommended)
+```properties
+# Resolves to latest patch version automatically
+jdkVersion=17          # → 17.0.14+7 (latest 17.x) - LTS
+jdkVersion=21          # → 21.0.7+7 (latest 21.x) - LTS
+jdkVersion=22          # → 22.0.2+7 (latest 22.x) - Non-LTS (warning shown)
+```
+
+**LTS vs Non-LTS Versions:**
+- **LTS (Long Term Support)**: 8, 11, 17, 21, 25 (Sept 2025), 29, 33...
+- **Non-LTS**: 9, 10, 12-16, 18-20, 22-24, 26-28...
+- **Recommendation**: Use LTS versions for production, or direct URLs for non-LTS
+
+### Full Version (Specific)
+```properties
+# Use exact version if needed
+jdkVersion=17.0.14     # → Exact version 17.0.14
+jdkVersion=21.0.6      # → Exact version 21.0.6
+jdkVersion=22.0.1      # → Exact version 22.0.1
+```
+
+### Version Examples by Distribution
+```properties
+# Temurin (Eclipse Adoptium) - default
+jdkVersion=17          # → 17.0.14+7-tem
+jdkVersion=21          # → 21.0.7+7-tem
+
+# Oracle OpenJDK
+jdkVersion=22          # → 22.0.2-oracle
+jdkDistribution=oracle_open_jdk
+
+# Amazon Corretto
+jdkVersion=17          # → 17.0.15-amzn
+jdkDistribution=corretto
+```
+
+**Note**: Major version resolution (e.g., `17` → `17.0.14+7`) is recommended as it automatically provides the latest security patches and bug fixes.
+
+## Supported JDK Distributions
+
+The Maven Wrapper supports 34+ JDK distributions through the Foojay Disco API using native distribution names:
+
+### Popular Distributions
+
+| Distribution | Description | Recommended Use |
+|-------------|-------------|-----------------|
+| `temurin` | Eclipse Adoptium (recommended) | General purpose, excellent support |
```

Comment:
Is it appropriate to recommend a specific distribution?

Reply:
Definitely no, I have switched that to `default` rather than `recommended`, as we need one.  But it came back ;-)

Reply:
FYI, the PR description also states Temurin is recommended.
Dunno if you want to update that too?